### PR TITLE
Correction of LDAP AuthN documentation

### PR DESCRIPTION
**Pull request by PetrGasparik**
- names of properties,
- $cas.version->$project.version,
- ldap.trustedCert should be in sample cas.properties file
- startTLS should be set to false, because sample config is for ldap://

### CODE REVIEW

File: cas-server-documentation/installation/LDAP-Authentication.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ LDAP integration is enabled by including the following dependency in the Maven W
 <dependency>
      <groupId>org.jasig.cas</groupId>
      <artifactId>cas-server-support-ldap</artifactId>
-     <version>${cas.version}</version>
+     <version>${project.version}</version>
```
Comment (mmoayyed) [inline cas-server-documentation/installation/LDAP-Authentication.md]: This should be reverted. Project version implies that it should the version of the maven overlay. We want the cas version here stated explicitly. and it's something that all other docs use as well. 

Otherwise, gorgeous. Thank you.

Comment (PetrGasparik) [inline cas-server-documentation/installation/LDAP-Authentication.md]: with $cas.version, it does not compile, saying that there is no such version.

what is supposed action now? I redo it and make another PR?

Comment (dima767) [inline cas-server-documentation/installation/LDAP-Authentication.md]: Just make the changes and push new commits to your fork's `4.1.x` branch. The updated commits will show up in this PR automatically.

Comment (mmoayyed) [inline cas-server-documentation/installation/LDAP-Authentication.md]: @PetrGasparik It wont compile because that is expected of you to define. See https://github.com/Jasig/cas-overlay-template/blob/master/pom.xml#L79

Comment (PetrGasparik) [inline cas-server-documentation/installation/LDAP-Authentication.md]: ok.

**Comment by PetrGasparik** [comment]
This is not linked to #1321, why?

I would like to make corrections to master doc also, but I don't know the URL where I can see current master wiki. Or, if there is none, how to visualize the .md file?

**Comment by mmoayyed** [comment]
#1321 is also a documentation issue but it does not deal with LDAP AuthN. It's a separate issue, that mostly deals with the upgrade guide, overseen by @robertoschwald . Your here seems to deal with something different. 

The master doc is in the master branch under cas-server-documentation. Find the Ldap doc in the installation folder, though I am sure you'll find that version is quite a bit different from the 4.1.x branch. So be careful. 

There are a number of ways to see the master wiki. You can:

1. Use a markdown editor to edit the .md files. Unfortunately, most editors do not do a good job of rendering code snippets via the github format. There is Atom, Sublime, and MarkdownPad. Use what works for you. 
2. One easy way to see the cas documentation visualized is by checking out the gh-pages branch, and building it locally. See the readme here https://github.com/Jasig/cas/tree/gh-pages

You should:
1. Make the changes there, build the site, and see it
2. Move the changes over to the master branch of the documentation.
